code/datums/wounds/robotic/robotic_blunt.dm

```diff
@@ -0,0 +1,1027 @@
+	if (!welder.tool_use_check())
+		return TRUE
+
+	var/their_or_other = (user == victim ? "their" : "[user]'s")
+	if (user)
+		user.visible_message(span_notice("[user] carefully holds [welder] to [their_or_other] [limb.plaintext_zone], slowly heating it..."))
+
+	var/delay_mult = 1
+	if (HAS_TRAIT(src, TRAIT_WOUND_SCANNED))
+		delay_mult *= 0.75
+
+	if (!welder.use_tool(target = victim, user = user, delay = 10 SECONDS * delay_mult, volume = 50, extra_checks = CALLBACK(src, PROC_REF(still_exists))))
+		return TRUE
+
+	var/wound_path = /datum/wound/burn/robotic/overheat/severe
+	if (user != victim && user.combat_mode)
+		wound_path = /datum/wound/burn/robotic/overheat/critical
+		user.visible_message(span_warning("[user] heats [victim]'s [limb.plaintext_zone] aggressively, overheating it far beyond the necessary point!"))
+		limb.receive_damage(burn = 10, damage_source = welder)
+
+	var/datum/wound/burn/robotic/overheat/overheat_wound = new wound_path
+	overheat_wound.apply_wound(limb, wound_source = welder)
+
+	to_chat(victim, span_green("Your [limb.plaintext_zone] is now heated, allowing it to be molded! Your next step is to have someone physically reset the superstructure with their hands."))
+	return TRUE
+
+// An RCD can be used on a T3 wound to finish its 1st treatment step with little risk and no burn wound
+/datum/wound/blunt/robotic/critical/proc/rcd_superstructure(obj/item/construction/rcd/treating_rcd, mob/user)
+	if (!treating_rcd.tool_use_check() || treating_rcd.get_matter(user) < ROBOTIC_T3_BLUNT_WOUND_RCD_COST)
+		return TRUE
+
+	if (user)
+		user.visible_message(span_notice("[treating_rcd] whirs to life as it begins replacing the damaged superstructure of [victim]'s [limb.plaintext_zone]..."))
+
+	var/delay_mult = 1
+	if (HAS_TRAIT(src, TRAIT_WOUND_SCANNED))
+		delay_mult *= 0.75
+
+	if (!treating_rcd.use_tool(target = victim, user = user, delay = 10 SECONDS * delay_mult, volume = 50, extra_checks = CALLBACK(src, PROC_REF(still_exists))))
+		return TRUE
+	treating_rcd.useResource(ROBOTIC_T3_BLUNT_WOUND_RCD_COST, user)
+
+	var/chance = 100
+	if (victim == user)
+		chance *= 0.75
+
+	if (HAS_TRAIT(user, TRAIT_KNOW_ROBO_WIRES))
+		chance *= 5
+	if (HAS_TRAIT(user, TRAIT_KNOW_ENGI_WIRES))
+		chance *= 2
+	if (HAS_TRAIT(user, TRAIT_DIAGNOSTIC_HUD))
+		chance *= 2
+	if (HAS_TRAIT(src, TRAIT_WOUND_SCANNED))
+		chance *= 2
+
+	if (prob(chance))
+		if (user)
+			user.visible_message(span_green("[treating_rcd] lets out a small ping as it finishes replacing the superstructure of [victim]'s [limb.plaintext_zone]."))
+		to_chat(victim, span_green("[treating_rcd] has finished replacing your [limb.plaintext_zone]'s superstructure! Your next step is to secure it with a screwdriver/wrench, though bone gel would also work."))
+		set_superstructure_status(TRUE)
+	else
+		if (user)
+			user.visible_message(span_warning("[user] screws up and accidentally damages more than they replaced with [treating_rcd]!"))
```

Comment:
Hardcoded pronouns, inconsistent messaging and spans

Reply:
forgot we had pronoun procs